resources/qml/TableView.qml

```diff
@@ -50,57 +52,106 @@ Item
                     anchors.leftMargin: UM.Theme.getSize("default_margin").width
                     anchors.right: parent.right
                     anchors.rightMargin: UM.Theme.getSize("narrow_margin").width
+                    anchors.verticalCenter: parent.verticalCenter
                     wrapMode: Text.NoWrap
                     text: modelData
                     font: UM.Theme.getFont("medium_bold")
                     elide: Text.ElideRight
                 }
-                Item
+
+                MouseArea
                 {
-                    //Resize handle.
+                    // Resize handle
                     anchors
                     {
                         right: parent.right
                         top: parent.top
                         bottom: parent.bottom
                     }
-                    width: UM.Theme.getSize("default_lining").width
+                    anchors.rightMargin: -width / 2
+                    width: UM.Theme.getSize("wide_lining").width * 2
+                    enabled: index < headerRepeater.count - 1
+                    acceptedButtons: Qt.LeftButton
+                    cursorShape: enabled ? Qt.SizeHorCursor : Qt.ArrowCursor
+
+                    property var dragLastPos
 
-                    MouseArea
+                    onPressed: (mouse) => { dragLastPos = mapToItem(parent, mouse.x, mouse.y) }
+
+                    onPositionChanged: (mouse) =>
```

Comment:
I didn't know this notation works! We usually just put `function(arg) { blah = bloeh(bleh) }` instead.

Reply:
It is quite uncommon indeed, but I discovered it somewhere in the Cura code :smile: not sure the other notation has the same meaning for callbacks, I'll try

Reply:
It's fine if this works! I'm not even sure we have a coherent standard for QML tbh 😅